tests/pytest/test_runner.py

```diff
@@ -123,7 +126,10 @@ def test_missing_libpython(monkeypatch):
         hdl_sources = [os.path.join(tests_dir, "designs", "runner", "runner.vhdl")]
         gpi_interfaces = [os.getenv("VHDL_GPI_INTERFACE", None)]
 
-    sim_tool = os.getenv("SIM", "icarus")
+    sim_tool = os.getenv(
+        "SIM",
+        "icarus" if os.getenv("HDL_TOPLEVEL_LANG", "verilog") == "verilog" else "ghdl",
```

Comment:
GHDL has subpar support for cocotb. We use NVC by default in CI now, so that's probably a better default for VHDL.
```suggestion
        "icarus" if os.getenv("HDL_TOPLEVEL_LANG", "verilog") == "verilog" else "nvc",
```

Reply:
Fixed